src/components/ui/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+import getMonthName from "../../utils/getMonthName";
+
+export default function Card({ ...props }) {
+  const { title, urlToImage, publishedAt } = props;
+  const date = new Date(publishedAt);
+  return (
+    <Link to={`/article/:${title}`} state={props}>
+      <div className="cursor-pointer group">
+        <img
+          src={urlToImage}
+          className="w-full h-44 object-cover rounded-sm"
+          alt={title}
+        />
+        <div className="flex gap-2 items-center text-gray-500 text-sm font-light mt-4 mb-2">
+          <span>
+            {getMonthName(date.getMonth())} {date.getDay()}
```

Comment:
You can create a helper function for formatting the date, separating this logic from the component